src/helm/benchmark/static/schema_classic.yaml

```diff
@@ -802,7 +814,7 @@ models:
     access: limited
     release_date: 2023-06-13
   - name: openai/gpt-4-1106-preview
-    display_name: openai/gpt-4-1106-preview
+    display_name: gpt-4-1106-preview
```

Comment:
For PaLM 2, we have a more user friendly name - should we do that here?  `GPT-4 (1106 preview)`?